R/tab_footnote.R

```diff
@@ -86,37 +86,38 @@ set_footnote <- function(loc, data, footnote) {
   UseMethod("set_footnote")
 }
 
-set_footnote.cells_data <- function(loc, data, footnote) {
+set_footnote.cells_title <- function(loc, data, footnote) {
 
-  resolved <- resolve_cells_data(data = data, object = loc)
+  if ((loc$groups %>% as.character())[-1] == "title") {
 
-  cols <- resolved$columns
-  rows <- resolved$rows
+    attr(data, "footnotes_df") <-
+      add_location_row(
```

Comment:
`add_location_row_footnotes`

(and other method calls below)

Reply:
Changed all the calls over to `add_location_row_footnotes()` in that file.